### PR TITLE
Issue #7345

### PR DESCRIPTION
**Pull request by get-satyam**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes I have read all the contributing guidelines on Pull requests

### Description

This program gives the no of points inside a circle that is described by its cartesian coordinates in a vector. The points lying on the border of the circle are a part of that circle too.

### Checklist

- [ ✅ ] I've read the contribution guidelines.
- [ ✅ ] I've checked the issue list before deciding what to submit.
- [ ✅ ] I've edited the `README.md` and link to my code.

### CODE REVIEW

**Comment by get-satyam** [comment]
@bhav09 please review my PR

File: C-Plus-Plus/cp/No_of_points_inside_a_circle.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*You are given an array points where points[i] = [xi, yi] is the coordinates of the ith point on a 2D plane. 
+Multiple points can have the same coordinates.
+You are also given an array queries where queries[j] = [xj, yj, rj] describes a circle centered at (xj, yj) 
+with a radius of rj.
+For each query queries[j], compute the number of points inside the jth circle. Points on the border of the 
+circle are considered inside.
+Return an array answer, where answer[j] is the answer to the jth query.*/
+
+#include<iostream>
+#include <vector>
```
Comment (siddharth25pandey) [inline C-Plus-Plus/cp/No_of_points_inside_a_circle.cpp]: Use single header file **bits/stdc++**

File: C-Plus-Plus/cp/No_of_points_inside_a_circle.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        for(int j = 0; j < points.size(); j++){
+		if((x-points[j][0])*(x-points[j][0])+(y-points[j][1])*(y-points[j][1]) <= r * r)    
+				ans[i]++;
+		}
+        cout<<ans[i]<<" ";            
+	}
+	return ans;
+}
+int main(){
+    vector<vector<int>>points{{1,3},{3,3},{5,3},{2,2}};
```
Comment (siddharth25pandey) [inline C-Plus-Plus/cp/No_of_points_inside_a_circle.cpp]: Use Dynamic Input

File: C-Plus-Plus/cp/No_of_points_inside_a_circle.cpp [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+     
+    vector<vector<int>>queries{{2,3,1},{4,3,1},{1,1,2}};
```
Comment (siddharth25pandey) [inline C-Plus-Plus/cp/No_of_points_inside_a_circle.cpp]: Use Dynamic Input

File: C-Plus-Plus/cp/No_of_points_inside_a_circle.cpp [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    countPoints(points, queries);
```
Comment (siddharth25pandey) [inline C-Plus-Plus/cp/No_of_points_inside_a_circle.cpp:51]: Display a message in print form as what are you showing output to the user

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
Do the suggested changes

File: C-Plus-Plus/cp/.vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+    "C_Cpp.errorSquiggles": "Disabled"
+}
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/.vscode/settings.json:3]: Remove this json file

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
- Add sample I/O at the end of the Code
- Add Time and Space Complexity at the end of the Code   
- Add a new line at the EOF
- Use proper display message how you are reading and writing outputs
- Add input-output prompts to make the program more interactive.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Readme missing

**Comment by get-satyam** [comment]
@ankitaggarwal23  I have done the necessary changes. Please see to it.

**Review by ankitaggarwal23 — COMMENTED** [review]
Don't push more than one code file in a single PR.

**Pull request closed without merging**